components/editor/CodeBlock.tsx

```diff
@@ -0,0 +1,583 @@
+/* eslint-disable unused-imports/no-unused-vars */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-useless-escape */
+import lowlight from 'lowlight';
+import { useState } from 'react';
+import Highlight from 'react-syntax-highlighter';
+
+import Caption from '../Caption';
+import IconClipboard from '../icons/Clipboard';
+
+interface CodeBlockProps {
+  children: string;
+  codeBlocks?: {code: string; title?: string; language?: string}[];
+  className?: string;
+  highlightClassName?: string;
+  highlightedLines?: number[];
+  language?: string;
+  hasWindow?: boolean;
+  showCopy?: boolean;
+  showCaption?: boolean;
+  caption?: string;
+  showLineNumbers?: boolean;
+  startingLineNumber?: number;
+  textSizeClassName?: string;
+  title?: string;
+}
+
+interface Theme {
+  [key: string]: {
+    display?: string;
+    background?: string;
+    color?: string;
+    fontWeight?: string | number;
+    backgroundColor?: string;
+    fontStyle?: string;
+    textDecoration?: string;
+  }
+}
+
+type Literal = 'true' | 'false' | 'yes' | 'no' | 'null';
+
+interface ValueContainer {
+  end: string;
+  endsWithParent: boolean;
+  excludeEnd: boolean;
+  contains: string[];
+  keywords: Literal[];
+  relevance: number;
+}
+
+interface Object {
+  begin: string;
+  end: string;
+  contains: ValueContainer[];
+  illegal: string;
+  relevance: number;
+}
+
+interface Array {
```

Comment:
Don't specify the name of the interface as a common type in TS. Rename it to something specific for this file.

Reply:
This code was removed on removal of lowlight.